ci/deploy.yml

```diff
@@ -69,3 +69,14 @@ suite-native deploy dev:
     - rsync --delete -va app-release.apk "${DEPLOY_DIRECTORY}/"
   tags:
     - deploy
+
+# Messaging system deploy to dev
+msg-system deploy dev:
+  stage: deploy to dev
+  needs:
+    - "msg-system config sign dev"
+  when: manual
```

Comment:
This could be automatic. Not sure if we want it though. Or only for `develop`? But since releasing new versions is not common I think we can keep it manual. What do you think @tomasklim?

Reply:
I think that we can keep it manual